### DISCUSSION TITLE
Very difficult to get questions answered

### DISCUSSION THREAD

**Discussion by RonBarkan** [micronaut-projects/micronaut-core#10849] [General]
Most questions I've been asking got no traction no matter where I posted them. This makes it difficult to use Micronaut.

I think community interaction is something you want, because it helps your project become better, address more needs and get new people to adopt it. However, if questions are not answered, the process gets stuck and people get frustrated, get a negative opinion on the project and stop using it / don't adopt it, which creates a negative momentum for the project.

Are there any plans to make it easier to get questions answered?

**Comment by graemerocher** [comment]
Which questions in particular?

**Comment by RonBarkan** [comment]
Right now, what frustrates me and I would really appreciate an answer to, is:  
https://github.com/micronaut-projects/micronaut-core/discussions/10787
